main.py

```diff
@@ -5,8 +5,8 @@
 
 import getPos
 #import sim.solarSystem
-import sim.spiralSystem
-#import sim.galaxyCollison
+#import sim.spiralSystem
+import sim.galaxyCollison
 #import sim.GalaxyCollison2
 #import renderMatplot
 import renderPygame
```

Comment:
Really? Need I say more?

Reply:
Yeah I didn't focus too much on the python aspect in this branch, although I think in main I might've cleared this up but maybe not commited?

Reply:
Well commit it to the branch then!